adaptive-education-ai/src/routes/IbmWatsonRouter.ts

```diff
@@ -0,0 +1,90 @@
+import {Router, Request, Response, NextFunction} from 'express';
+import SemanticRolesModel from '../ibm-watson/SemanticRolesModel';
+import IbmWatsonNluService from '../ibm-watson/IbmWatsonNluService';
+const fs = require('fs');
+const NaturalLanguageUnderstandingV1 = require('watson-developer-cloud/natural-language-understanding/v1.js');
+const ibmWatsonNluService = new IbmWatsonNluService();
+
+export class IbmWatsonRouter {
+  router: Router;
+
+  /**
+   * Initialize the Router
+   */
+  constructor() {
+    this.router = Router();
+    this.init();       
+  }
+
+  /**
+   * Get
+   */
+  public getWatsonResults(req: Request, res: Response, next: NextFunction) {
+    res.send(ibmWatsonNluService.getWatsonResult());    
+  }
+
+  /**
+   * Get
+   */
+  public getSemanticRolesResult(req: Request, res: Response, next: NextFunction) {
+    res.send(ibmWatsonNluService.getSemanticRolesResult());    
+  }  
+
+  /**
+   * Get
+   */
+  public getSaoQuestions(req: Request, res: Response, next: NextFunction) {
+    res.send(ibmWatsonNluService.generateSaoQuestions());    
+  }
+
+  /**
+   * POST Analyze-Text.
+   */
+  public analyzeText(req: Request, res: Response, next: NextFunction) {
+    let text = req.body.text;
+    let watsonResult = '';
+
+    // setting up natural language understanding
+    let nlu = new NaturalLanguageUnderstandingV1({
+        username: 'af6311ab-9757-48bc-8e4f-7598198ce369',

```

Comment:
Are these credentials something that we should remove from the repo and pass around the team outside of source control? (Of course, we'd have to generate new ones now, since they're in the history)